rubygem-katello/rubygem-katello.spec

```diff
@@ -95,6 +95,14 @@ Summary:    Documentation for rubygem-%{gem_name}
 %description doc
 This package contains documentation for rubygem-%{gem_name}.
 
+%package ostree
```

Comment:
I think we need to custom name this as RPM will try to use `-` to connect, and we want to use the more typical trend of `_`:

```tfm-rubygem-katello-ostree```

vs

```tfm-rubygem-katello_ostree```